includes/class-wcpdf-admin.php

```diff
@@ -515,16 +515,19 @@ public function data_input_box_content( $post_or_order_object ) {
 			// data
 			$data = array(
 				'number' => array(
-					'label'  => __( 'Invoice Number:', 'woocommerce-pdf-invoices-packing-slips' ),
+					'label' => __( 'Invoice Number:', 'woocommerce-pdf-invoices-packing-slips' ),
 				),
 				'date'   => array(
-					'label'  => __( 'Invoice Date:', 'woocommerce-pdf-invoices-packing-slips' ),
+					'label' => __( 'Invoice Date:', 'woocommerce-pdf-invoices-packing-slips' ),
 				),
 				'display_date' =>  array(
-					'label'  => __( 'Display Date:', 'woocommerce-pdf-invoices-packing-slips' ),
+					'label' => __( 'Invoice Display Date:', 'woocommerce-pdf-invoices-packing-slips' ),
+				),
+				'creation_status' =>  array(
+					'label' => __( 'Invoice Created:', 'woocommerce-pdf-invoices-packing-slips' ),
```

Comment:
I would rename this to 'Invoice Created Via'. Also perhaps we should drop the title case for these labels? Especially since we now have labels with more than two words, I think:

```
Invoice number:
Invoice date:
Invoice display date:
Invoice created via:
``` 

looks better than:

```
Invoice Number:
Invoice Date:
Invoice Display Date:
Invoice Created Via:
```

What do you think @alexmigf, @YordanSoares, @aparnascodex?

Reply:
I agree.

Reply:
I'm also agreed.

Reply:
What about just?

```
Number:
Date:
Display Date:
Created Via:
```

Reply:
I like that visually, but I wonder if there is a downside from an accessibility point of view. For instance people that are vision impaired and use a screen reader. Perhaps we can leave it for now.